docs/content/guides/author-apps/kubernetes/howto-yaml-base-support/index.md

```diff
@@ -0,0 +1,83 @@
+---
+type: docs
+title: "How-To: Migrate from existing Kubernetes resources"
+linkTitle: "Migrate from YAML"
+description: "Learn how to migrate to a Radius container from existing Kubernetes YAML"
+weight: 500
+categories: "How-To"
+tags: ["containers","Kubernetes"]
+---
+
+This guide will provide an overview of how to migrate your existing Kubernetes resources from [Kubernetes YAML](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#writing-a-deployment-spec).
+
+### Prerequisites
+
+Before you get started, you'll need to make sure you have the following tools and resources:
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [rad CLI]({{< ref getting-started >}})
+- [Radius initialized with `rad init`]({{< ref howto-environment >}})
+
+### Step 1: Define a base Kubernetes YAML file
+
+In a file named `kubernetes.yaml` define all the Kubernetes resources that correspond to the new Radius container.
+
+Note that the names of the Deployments, Services, and Config Maps all must match each other and the name of the new Radius container. Any other resources can be named differently.
+
+{{< rad file="snippets/basemanifest.yaml" embed=true lang="yaml">}}
+
+### Step 2: Define a Radius container
+
+[Radius containers]({{< ref "/guides/author-apps/containers/overview" >}}) represent a containerized workload within your Radius Application. You can define a Radius container and reference your Kubernetes YAML in order to deploy and manage it within Radius.
+
+In a file named "app.bicep" add a new Container resource, specifying:
+
+- The container image
```

Comment:
It's kind of confusing how the user needs to specify the container image again. I'd see if there's a way to explain this a little more, and why it's required to specify again.